virttest/utils_time.py

```diff
@@ -0,0 +1,258 @@
+import logging
+import re
+
+from avocado.utils import process
+from avocado.core import exceptions
+
+from virttest import error_context
+from virttest import utils_disk
+
+
+@error_context.context_aware
+def get_host_timezone():
+    """
+    Get host's timezone.
+    """
+    timezone_cmd = 'timedatectl | grep "Time zone"'
+    timezone_pattern = '^(?:\s+Time zone:\s)(\w+\/\S+|UTC)(?:\s\(\S+,\s)([+|-]\d{4})\)$'
+    error_context.context("Get host's timezone", logging.info)
+    host_timezone = process.system_output(timezone_cmd, timeout=240, shell=True)
+    try:
+        host_timezone_set = re.match(timezone_pattern, host_timezone).groups()
+        return {"timezone_city": host_timezone_set[0],
+                "timezone_code": host_timezone_set[1]}
+    except AttributeError, IndexError:
```

Comment:
has to be `except (AttributeError, IndexError):`, and the rest of this kind of usage.

Reply:
Thanks, modified it.